src/main/java/ca/uhn/fhir/jpa/starter/common/StarterJpaConfig.java

```diff
@@ -448,5 +462,45 @@ public static IServerConformanceProvider<?> calculateConformanceProvider(IFhirSy
 			throw new IllegalStateException();
 		}
 	}
+
+	private void registerLocationHierarchyTypes(RestfulServer restfulServer, DaoRegistry daoRegistry) {
+		IFhirResourceDao<Location> locationIFhirResourceDao = daoRegistry.getResourceDao("Location");
+		LocationHierarchyResourceProvider locationHierarchyResourceProvider = new LocationHierarchyResourceProvider();
+		locationHierarchyResourceProvider.setLocationIFhirResourceDao(locationIFhirResourceDao);
+
+		restfulServer.registerProvider(locationHierarchyResourceProvider);
+		restfulServer.getFhirContext().registerCustomType(LocationHierarchy.class);
+		restfulServer.getFhirContext().registerCustomType(LocationHierarchyTree.class);
+		restfulServer.getFhirContext().registerCustomType(Tree.class);
+		restfulServer.getFhirContext().registerCustomType(ParentChildrenMap.class);
+		restfulServer.getFhirContext().registerCustomType(SingleTreeNode.class);
+		restfulServer.getFhirContext().registerCustomType(TreeNode.class);
+		restfulServer.getFhirContext().registerCustomType(ChildTreeNode.class);
+	}
+
+	private void registerPracitionerDetailsTypes(RestfulServer restfulServer, DaoRegistry daoRegistry) {
+		IFhirResourceDao<Practitioner> practitionerIFhirResourceDao = daoRegistry.getResourceDao("Practitioner");
+		IFhirResourceDao<PractitionerRole> practitionerRoleIFhirResourceDao = daoRegistry.getResourceDao("PractitionerRole");
+		IFhirResourceDao<CareTeam> careTeamIFhirResourceDao = daoRegistry.getResourceDao("CareTeam");
+		IFhirResourceDao<OrganizationAffiliation> organizationAffiliationIFhirResourceDao = daoRegistry.getResourceDao("OrganizationAffiliation");
+		IFhirResourceDao<Organization> organizationIFhirResourceDao = daoRegistry.getResourceDao("Organization");
+		IFhirResourceDao<Location> locationIFhirResourceDao = daoRegistry.getResourceDao("Location");
```

Comment:
Add the Hardcoded strings to Constants or use the existing Constants.

Reply:
Done